bndtools.core/src/bndtools/editor/project/RunRequirementsPart.java

```diff
@@ -474,9 +512,7 @@ protected boolean performSelectionDrop(ISelection data, Object target, int locat
             }
 
             if (!adding.isEmpty()) {
```

Comment:
This was on purpose. The intention was to remove the empty-check, but it was still necessary in the DropAdapter due to the boolean-return.

Going to add the check above for consistency.